docs/tutorials/timeseries/forecasting-indepth.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<p>When should users employ this approach if it is not advisable? Are there any advantages to using it over option 1?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/autogluon/autogluon/pull/3297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
let's remove it then